invitation/client.go

```diff
@@ -76,6 +76,42 @@ func (c *Client) Create(ctx context.Context, params *CreateParams) (*clerk.Invit
 	return invitation, err
 }
 
+type BulkCreateParams struct {
+	clerk.APIParams
+	Invitations []*CreateParams
+}
+
+func (b BulkCreateParams) MarshalJSON() ([]byte, error) {
```

Comment:
:question: Why do we need this custom marshaling method?

Reply:
We need the custom marshaling because the API's request for bulk is an array of regular requests, so it’s not bound to a key like `{"invitations": [...]}`. To be accurate in SDK, the correct type would be `type BulkCreateParams []*CreateParams`. However this wouldn’t be compatible with `req.SetParams`.

Do you have a better alternative to suggest?

Reply:
Thanks for explaining! It is indeed a problem with the current implementation.

> Do you have a better alternative to suggest?

Not really. :disappointed:  As discussed offline, let's settle on the exported types for now and solidify the API.

Reply:
I’ve refactored the types as we discussed a bit [here](https://github.com/clerk/clerk-sdk-go/pull/351/commits/0d2d9a136df95219204bcf47d7942ce0bfda3a99). Please take another look when you get a chance.